src/main/java/Dan.java

```diff
@@ -0,0 +1,172 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+enum TASK_TYPE {TODO, DEADLINE, EVENT}
+public class Dan {
+
+    private final static String greets = "\nDan: \n";
+    private static MyList tasks = new MyList(100);
+    private final static String[] commands = new String[] {
+            "toDo [TASK]",
+            "deadline [TASK] /by [DEADLINE]",
+            "event [TASK] /from [START_TIME] /to [END_TIME]",
+            "list", "bye",
+            "mark [TASK_ID]", "unmark [TASK_ID]"
+    };
+
+
+    public static void main(String[] args) {
+        hello();
+        chat();
+        goodbye();
+    }
+
+
+
+    private static void chat() {
+        String text;
+        String[] texts;
+        String command;
+        a: while (true) {
+            try {
+            text = new Scanner(System.in).nextLine();
+            texts = text.split(" ");
+            command = texts[0].toLowerCase();
+            switch (command) {
+                case "bye":
+                    break a;
```

Comment:
perhaps it would be better if you can extract these 'instructions' out to a Ui class